src/stepper/Stepper.tsx

```diff
@@ -73,12 +76,16 @@ export default class Stepper extends PureComponent<StepperProps, StepperState> {
 
   constructor(props: StepperProps) {
     super(props)
-    this.ref = React.createRef()
+    this.valueElementRef = React.createRef()
+    this.containerRef = React.createRef()
   }
 
   componentDidMount() {
     this.handleFontSize()
     window.addEventListener('resize', this.handleFontSize.bind(this))
+    if (this.containerRef.current) {
+      this.containerRef.current.focus()
```

Comment:
Do we always want to focus when the component is mounted?

Reply:
Shouldn't this be done by the overlay?

Reply:
No my mistake I'll add the the condition on the focus prop